src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -0,0 +1,69 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+// ------------------------------------------------------------------------------
+// Changes to this file must follow the http://aka.ms/api-review process.
+// ------------------------------------------------------------------------------
+
+namespace System.Text.Json
+{
+    public enum JsonCommentHandling : byte
+    {
+        AllowComments = (byte)1,
+        Default = (byte)0,
+        SkipComments = (byte)2,
+    }
+    public partial class JsonReaderException : System.Exception
+    {
+        public JsonReaderException(string message, long lineNumber, long lineBytePosition) { }
+        public long LineBytePosition { get { throw null; } }
+        public long LineNumber { get { throw null; } }
+    }
+    public partial struct JsonReaderOptions
+    {
+        private int _dummyPrimitive;
+        public System.Text.Json.JsonCommentHandling CommentHandling { get { throw null; } set { } }
+    }
+    public partial struct JsonReaderState
+    {
+        private object _dummy;
+        private int _dummyPrimitive;
+        public JsonReaderState(int maxDepth = 64, System.Text.Json.JsonCommentHandling commentHandling = System.Text.Json.JsonCommentHandling.Default) { throw null; }
+        public long BytesConsumed { get { throw null; } }
+        public int MaxDepth { get { throw null; } }
+        public System.Text.Json.JsonReaderOptions Options { get { throw null; } }
```

Comment:
Why does the ctor take a JsonCommentHandling but JsonReaderOptions is exposed as the get-only property?  Shouldn't the ctor take a JsonReaderOptions instead of a JsonCommentHandling?